src/open_inwoner/accounts/tests/test_contact_views.py

```diff
@@ -275,10 +290,69 @@ def test_adding_contact_with_invalid_last_name_chars_fails(self):
                             "Please make sure your input contains only valid characters "
                             "(letters, numbers, apostrophe, dash, space)."
                         )
-                    ]
+                    ],
                 }
                 self.assertEqual(response.context["form"].errors, expected_errors)
 
+    #
+    # Contacts with duplicate emails
+    #
+    @override_settings(
+        AUTHENTICATION_BACKENDS=["digid_eherkenning.backends.DigiDBackend"],
+    )
+    @patch("digid_eherkenning.backends.DigiDBackend.authenticate")
+    def test_digid_contact_with_duplicate_email_success(self, m):
+        m.return_value = self.digid_user
+
+        existing_user = DigidUserFactory(
+            first_name="Luke",
+            last_name="Skywalker",
+            bsn="111111111",
+            email=self.digid_user.email,
+        )
+
+        response = self.app.get(self.create_url, user=self.user)
+
+        form = response.forms["contact-form"]
+        form["first_name"] = existing_user.first_name
+        form["last_name"] = existing_user.last_name
+        form["email"] = existing_user.email
+
+        response = form.submit(user=self.digid_user)
+        pending_invitation = self.digid_user.contacts_for_approval.first()
+
+        self.assertEqual(response.status_code, 302)
+        self.assertContains(response.follow(), existing_user.email)
```

Comment:
Webtest checks response codes for the thing we do, so a `.follow()` only works for a redirecting 3xx response. Similarly does a .get() or post() checks for a 2xx etc (you can specify an exact code as argument).